scvi/dataset/hemato.py

```diff
@@ -10,16 +10,16 @@ class HematoDataset(GeneExpressionDataset):
           "file=GSM2388072%5Fbasal%5Fbone%5Fmarrow%2Eraw%5Fumifm%5Fcounts%2Ecsv%2Egz"
     datazip_url = "https://github.com/romain-lopez/scVI-reproducibility/raw/master/additional/data.zip"
 
-    def __init__(self, unit_test=False):
-        self.save_path = 'data/HEMATO/' if not unit_test else 'tests/data/HEMATO/'
+    def __init__(self, save_path='data/HEMATO/'):
+        self.save_path = save_path
 
         # Originally: GSM2388072_basal_bone_marrow.raw_umifm_counts.csv.gz
 
         self.download_name = "bBM.raw_umifm_counts.csv.gz"
         self.gene_names_filename = "bBM.filtered_gene_list.paper.txt"
         self.spring_and_pba_filename = "bBM.spring_and_pba.csv"
 
-        if not unit_test and not os.path.isfile('data/data.zip'):
+        if not save_path == 'tests/data/HEMATO/' and not os.path.isfile('data/data.zip'):
```

Comment:
We don't have `data.zip` in unit test (since it's a folder that contains a subset of `HEMATO`, `PBMC` and `RETINA`. Is that okay if I add a dummy `data.zip` file in `tests/data/`?

Reply:
Yes, that seems like a good solution